server/src/metadata/migration-runner/migration-runner.service.ts

```diff
@@ -45,9 +45,10 @@ export class MigrationRunnerService {
 
     // Loop over each migration and create or update the table
     // TODO: Should be done in a transaction
-    flattenedPendingMigrations.forEach(async (migration) => {
+
+    for (const migration of flattenedPendingMigrations) {
```

Comment:
This is to make sure all handleTableChanges migrations are executed before going forward. Sometimes we were trying to insert seeds before creating all the columns due to how await behaves inside anonymous functions